Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -49,21 +50,25 @@ YUI.add('ez-objectrelationlistloadplugin', function (Y) {
                 actualRelatedContent.load(loadOptions, stack.add(function (error) {
                     if (error) {
                         e.target.set("loadingError", true);
+                        loadingError = true;
                     } else {
-                        relatedContentListArray.push(actualRelatedContent);
+                        if(loadedRelation.indexOf(value.destination) === -1) {
```

Comment:
CS: always a space after `if`

it's a bit too late for this condition, I mean it would be better to test that before instantiating and loading the content.